contracts/TokenSale.sol

```diff
@@ -43,71 +52,69 @@ contract TokenSale is FinalizableCrowdsale, Pausable {
      * @dev initialization function
      * @param _startTime The timestamp of the beginning of the crowdsale
      * @param _endTime Timestamp when the crowdsale will finish
-     * @param _whitelist contract containing the whitelisted addresses
-     * @param _starToken STAR token contract address
-     * @param _companyToken ERC20 contract address that has minting capabilities
-     * @param _tokenOwnerAfterSale Address that this TokenSale will pass the token ownership to after it's finished. Only works when TokenSale mints tokens, otherwise must be `0x0`.
-     * @param _rate The token rate per ETH
-     * @param _starRatePer1000 The token rate per 1/1000 STAR
-     * @param _wallet FundsSplitter wallet that redirects funds to client and Starbase.
+     * @param _externalAddresses contract containing the whitelisted addresses
```

Comment:
Fascinating. Good idea. Never seen it done like this. Well done

Reply:
Whops. The comment wasnt changed.